infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -96,6 +96,12 @@
     @Getter
     private long globalLogTail = Address.getMinAddress();
 
+    /**
+     * - {@link SequencerServer::globalSequence}: A simple counter that goes up on every request.
+     */
+    @Getter
+    private Long globalSequence = Address.getMinAddress() + 1;
```

Comment:
why +1?

Reply:
Multiple reasons:
1. If the min address is set to a value that was already returned, this avoids returning a counter value that was already returned before. 
2. Avoids returning 0 as a counter value (since 0 is the min address). This helps distinguish from initial state
3. easier for testing since we can use compare operator starting at 0.

Reply:
can you add this to the java doc?

Reply:
maybe even add this number into Address class?

Reply:
done.